src/pages/AddMembersPage/NewMemberList.test.tsx

```diff
@@ -0,0 +1,180 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { client } from 'lib/gql/client';
+
+import { TestWrapper } from 'utils/testing';
+
+import { Group } from './AddMembersPage';
+import NewMemberList from './NewMemberList';
+
+const save = async () => {
+  return [];
+};
+
+const group: Group = {
+  id: 1,
+  name: 'test',
+};
+
+jest.mock('lib/gql/client', () => ({
+  client: { query: jest.fn() },
+}));
+
+beforeEach(() => {
```

Comment:
Why do this `beforeEach`? Usually we do this afterEach to cleanup the mocks and not affect other tests that may run afterwrads. You setup the mock, and then clear it in the same test.